ims/profile.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+
+package ims
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// The user token is used to authorize the request and to define which user's profile is requested.
+func (c *Client) GetProfile(userToken string) (string, error){
```

Comment:
I know this might sound silly, but I prefer to have a struct for the request and the response of this API call.

A struct for the request would allow the library to add new input parameters (if any are introduced in the future) without breaking the users of the library.

A struct for the response would allow the library to parse the user profile (which you currently don't do) without breaking the users of the library. Probably figuring out which fields to parse is a bit too much research for now (I know the status of the documentation). For the moment, though, you can just return a response struct with the body.

In summary, it would be nice if you could change the API to:

```go
type GetProfileRequest struct {
	AccessToken string
}

type GetProfileResponse struct {
	Body []byte
}

func (c *Client) GetProfile(req *GetProfileRequest) (*GetProfileResponse, error) {
	...
}
```

Reply:
And done